src/main/frontend/components/page.cljs

```diff
@@ -147,11 +147,15 @@
       (when (seq queries)
         [:div#today-queries.mt-10
          (for [query queries]
-           (rum/with-key
-             (block/custom-query {:attr {:class "mt-10"}
```

Comment:
Weirdly enough, this component [has a catch-error](https://github.com/logseq/logseq/blob/acc5a62c94cc84121fd34b448d750cfbc2ca37b2/src/main/frontend/components/block.cljs#L2510-L2513) but it was not taking effect

Reply:
It seems that rum components don't catch errors in both :init and :will-component lifecycles.
https://github.com/tonsky/rum/blob/gh-pages/src/rum/core.clj#L131-L138

Reply:
It's likely that lifecycle hooks have something to do with it but wouldn't explain why the page component works while `custom-query` doesn't